tests/test_jobid.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+from os import environ
+
+from picas.jobid import add_job_id, remove_job_id
+from picas.documents import Document
+
+
+class TestJobid(unittest.TestCase):
+
+    def setUp(self):
+        self.doc = Document()
+        self.env_vars = {
+            "DIRACJOBID": "dirac_job_id",
+            "SLURM_JOB_ID": "slurm_job_id",
+            "GLITE_WMS_JOBID": "wms_job_id",
+            "CREAM_JOBID": "cream_job_id",
+            "PBS_JOBID": "pbs_job_id"
+        }
+
+    def test_add_job_id(self):
+        """ Test if job_id is added to token"""
+        for test in self.env_vars:
```

Comment:
I dont understand this for loop, it seems convoluted. What is going on here?
Can't you just loop over the `self.env_vars`, write those, and then check the token against the original values?

Reply:
Well, that is exactly what this loop does. It loops over self.env_vars keys, writes a string (for convenience chosen to be "http/jobid") into environmental variables, and then checks if add_job_id adds this string to the token. If you prefer another implementation, please make a suggestion.

Reply:
It makes sense now, I must have missed the add_job_id line.